File: 1323.cpp
/*
	Author: Zoljargal Gantumur
	Runtime: 0 ms, faster than 100.00% of C++ online submissions
	Memory Usage: 6.1 MB, less than 43.43% of C++ online submissions
*/

class Solution {
public:
    int maximum69Number (int num) {
        string s = to_string(num);
        for (int i=0; i<s.size(); i++){
            if(s[i]=='9') continue;
            else s[i]='9';
            break;
        }
        return stoi(s);
    }
};

File: 1323.py
# Author: Zoljargal Gantumur
# Runtime: 16 ms, faster than 99.90% of Python3 online submissions
# Memory Usage: 14 MB, less than 14.51% of Python3 online submissions

class Solution:
    def maximum69Number (self, num: int) -> int:
        s = str(num)
        i = s.find("6")
        if i == -1:
            return num
        return int(s[:i] + "9" + s[i +1 :])
    
